src/commands/create/create-config.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const path = require('path');
+
+exports.applyConfigInDirectory = function(directoryPath, packageData) {
+    let createConfig = getCreateConfig(directoryPath);
+    let createIgnore = createConfig.ignore || {};
+    let ignoredFiles = createIgnore.files;
+    let ignoredScripts = createIgnore.scripts;
+    let ignoredDependencies = createIgnore.dependencies;
+
+    if (ignoredScripts && packageData.scripts) {
+        removeKeys(packageData, 'scripts', ignoredScripts);
+    }
+
+    if (ignoredDependencies) {
+        if (packageData.dependencies) {
+            removeKeys(packageData, 'dependencies', ignoredDependencies);
+        }
+        if (packageData.devDependencies) {
+            removeKeys(packageData, 'devDependencies', ignoredDependencies);
+        }
+    }
+
+    if (ignoredFiles) {
+        ignoredFiles.forEach(file => {
+            fs.unlinkSync(path.resolve(directoryPath, file));
+        });
+    }
+}
+
+function getCreateConfig(directoryPath) {
+    let createConfigPath = path.resolve(directoryPath, '.createconfig');
```

Comment:
Just a thought, we could probably avoid a lot of work in this file by simply looking at a key in the `package.json` instead of a `.createconfig` file. What do you think about that?

**package.json**
```json
{
  "createConfig": {

   }
}
```